api/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.2.2 on 2023-06-16 15:39
+# Generated by Django 4.2 on 2023-07-09 12:30
```

Comment:
I think you forgot to run `poetry install` :)

Reply:
I did, but at some point downgraded to check something, right now I have 4.2.2 but I did not create a new commit because I think it won't brake anything and correct version will be back at next commit.

Reply:
nah, I was just being fastidious :) This is obviously not a major version change.